libs/core/src/components/sage-avatar/sage-avatar.scss

```diff
@@ -0,0 +1,74 @@
+:host {
+  /**
+    * @prop --size: Custom size of avatar. Setting this will *not* override a set `size` prop value.
+  */
+
+  --size: var(--sage-spacing-2xl);
+}
+
+$sage-avatar-sizes: (
+  xs: var(--sage-spacing-md),
+  sm: 32px,
+  md: 40px,
+  lg: 56px,
+  xl: var(--sage-spacing-2xl)
+);
+
+@each $size, $value in $sage-avatar-sizes {
+  :host(.sage-avatar--#{$size}) div {
+    height: $value;
+    width: $value;
+  }
+}
+
+:host(.sage-avatar--admin) div {
+  border-radius: 12px;
+}
+
+div {
+  --radius-round: 100px;
+
+  align-items: center;
+  background-color: var(--sage-color-primary-200);
+  block-size: var(--size);
+  border: var(--sage-border-default);
+  border-color: transparent;
```

Comment:
Is a border actually needed if it's set to be transparent?

Reply:
Thanks for pointing this out. For some reason, I was always under the impression that there needed to be a defined border in order for the radius to take effect, but apparently not. TIL.